### DISCUSSION TITLE
Hide "liked by" posts

### DISCUSSION THREAD

**Discussion by bopx** [AprilSylph/XKit-Rewritten#1267] [Suggestion Box]
It could be that I wasn't able to find it, but I would like to not see "liked by" posts. Not the posts I've liked but the posts that pop up trying to show me posts that other people have liked. Basically another algorithm recommended for you type of post. Thank you.

**Comment by marcustyphoon** [comment]
You can turn these on and off in Tumblr itself at the bottom of https://www.tumblr.com/settings/dashboard !

**Reply by bopx** [reply]
Thank you so much!! I totally missed that.
